docs/standard/analyzers/portability-analyzer.md

```diff
@@ -7,35 +7,66 @@ ms.assetid: 0375250f-5704-4993-a6d5-e21c499cea1e
-
-After running the analysis, you'll see your .NET Portability Report. Only types that are unsupported by a target platform appear in the list and you can review recommendations in the **Messages** tab in the **Error List**. You can also jump to problem areas directly from the **Messages** tab.
+### Analyze portability
+To analyze your entire project in Visual Studio, right-click on your project in **Solution Explorer** and select **Analyze Assembly Portability**. Otherwise, go to the **Analyze** menu and select **Analyze Assembly Portability**. From there, select your project’s executable or DLL.
 
-![Portability Report](./media/portability-analyzer/portability-report.png)
+![Portability Analyzer from Solution Explorer](./media/portability-analyzer/portability-solution-explorer.png)
 
-If you don’t want to use Visual Studio, you can use the Portability Analyzer from the command prompt. Just download the API Portability Analyzer from the [Microsoft/dotnet-apiport](https://github.com/Microsoft/dotnet-apiport/releases) repository.
+If you want to use [ApiPort console app](https://aka.ms/apiportdownload).
 
 * Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
 * To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
+* Run `\...\ApiPort.exe -?` to get more help
+
+### View and interpret portability result
+
+Only APIs that are unsupported by a Target Platform appear in the report. 
+After running the analysis in Visual Studio, you'll see your .NET Portability report file link pops up. If you used [ApiPort console app](https://aka.ms/apiportdownload), your .NET Portability report is saved as a file in the format you specified. The default is in an Excel file (*.xlsx*) in your current directory. 
+
+#### Portability Summary 
+
+![Portability Summary](./media/portability-analyzer/PortabilitySummary.PNG)
+
+The Portability Summary section of the report shows the the portability percentage for each assembly included in the run. In the previous example, 89.74% of the .NET Framework APIs used in the `ConsoleAppFramework` app are available in .NET Core + Platform Extensions v2.2. If you run the .NET Portability Analyzer tool against multiple assemblies, each assembly should have a row in the Portability Summary report.
 
-Your .NET Portability Report is saved as an Excel file (*.xlsx*) in your current directory. The **Details** tab in the Excel Workbook contains more information.
+#### Details
+
+![Portability Details](./media/portability-analyzer/PortabilityDetails.PNG)
+
+The Details section of the report lists the APIs missing from one of the Target Platforms. 
+
+ - Target type: the type has missing API from a Target Platform 
+ - Target member: the method is missing from a Target Platform 
+ - Header for assembly name entries: the .NET Framework assembly that the missing
```

Comment:
This header is a bug - there's an issue for it that we discussed and @Lxiamail was going to verify is already fixed (not sure what the status is). Needs to be ingested into the service

Reply:
Fixed the header